VAMobile/src/screens/HealthScreen/Pharmacy/PrescriptionHistory/PrescriptionHistoryNoMatches.tsx

```diff
@@ -3,45 +3,28 @@ import React, { FC } from 'react'
 
 import { Box, TextView, VAScrollView } from 'components'
 import { NAMESPACE } from 'constants/namespaces'
-import { PrescriptionHistoryTabConstants, PrescriptionHistoryTabs } from 'store/api/types'
 import { ViewStyle } from 'react-native'
 import { a11yLabelVA } from 'utils/a11yLabel'
 import { useTheme } from 'utils/hooks'
 
 type PrescriptionHistoryNoMatchesProps = {
-  /** currently selected tab */
-  currentTab: PrescriptionHistoryTabs
   /** whether a filter has been applied to the results */
   isFiltered: boolean
 }
 
-const PrescriptionHistoryNoMatches: FC<PrescriptionHistoryNoMatchesProps> = ({ currentTab, isFiltered }) => {
+const PrescriptionHistoryNoMatches: FC<PrescriptionHistoryNoMatchesProps> = ({ isFiltered }) => {
   const { t } = useTranslation(NAMESPACE.COMMON)
   const theme = useTheme()
 
-  const getFilteredTabString = () => {
-    switch (currentTab) {
-      case PrescriptionHistoryTabConstants.ALL:
-        return t('prescription.history.empty.filtered.all')
-      case PrescriptionHistoryTabConstants.PENDING:
-        return t('prescription.history.empty.filtered.pending')
-      case PrescriptionHistoryTabConstants.TRACKING:
-        return t('prescription.history.empty.filtered.tracking')
-    }
-  }
-
   const getFilteredNoMatch = () => {
-    let contenta11y = {}
-    if (currentTab === PrescriptionHistoryTabConstants.ALL) {
-      contenta11y = { accessibilityLabel: a11yLabelVA(t('prescription.history.empty.filtered.all')) }
-    }
+    const contenta11y = { accessibilityLabel: a11yLabelVA(t('prescription.history.empty.filtered.all')) }
     return (
       <>
         <TextView textAlign={'center'} variant="MobileBodyBold">
```

Comment:
I know this wasn't previously, but shouldn't this be a header?

Reply:
Make sense to me, added the accessibilityRole